proxy/defaultresponse_test.go

```diff
@@ -0,0 +1,26 @@
+package proxy_test
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/proxy/proxytest"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDefaultResponse(t *testing.T) {
+	p := proxytest.Config{
+		Routes: eskip.MustParse(`* -> <shunt>`),
+	}.Create()
+	defer p.Close()
+
+	rsp, body, err := p.Client().GetBody(p.URL)
+	require.NoError(t, err)
+
+	assert.Equal(t, http.StatusNotFound, rsp.StatusCode)
+	assert.Len(t, body, 0)
+	assert.Equal(t, "0", rsp.Header.Get("Content-Length"))
```

Comment:
This is an indirect test. It is hard to test that response is not chunked because http client handles chunked encoding transparently. To test it we would need to create net connection and send/receive raw HTTP request and response.

Reply:
I don't mind, but sounds simple enough to do:
```
GET / HTTP/1.1\r\n
Host: test\r\n
\r\n
```